web/templates/compose_banner/unmute_topic_banner.hbs

```diff
@@ -0,0 +1,10 @@
+{{#> compose_banner }}
+    <p>
+        {{#if (eq muted_narrow "stream")}}
+            {{#tr}}Your message was sent to a stream you have muted.{{/tr}}
+        {{else if (eq muted_narrow "topic")}}
+            {{#tr}}Your message was sent to a topic you have muted.{{/tr}}
+        {{/if}}
+        <span>{{#tr}}You will not receive notifications about new messages.{{/tr}}</span>
```

Comment:
I don't think we need a span around this; it can just be a next sentence in the paragraph?

Reply:
Removed this worked fine in my testing, so going with that to avoid a round-trip.